Makefile

```diff
@@ -15,9 +15,12 @@ help:
 	@echo "    push-check:         Replicates the github workflow checks as close as possible"
 	@echo "                        (do this before pushing!)"
 
+VERSION="NotUsed"
+COMMIT_HASH=$(shell git rev-parse HEAD)
+
 .PHONY: image-builder
 image-builder:
-	go build -o image-builder ./cmd/image-builder/
+	go build -ldflags "-X main.Version=$(VERSION) -X main.GitHash=$(COMMIT_HASH)" -o image-builder ./cmd/image-builder/
```

Comment:
go includes the git revision automatically during the build (if build from git) so you can just read it via `debug.ReadBuildInfo()`. See https://github.com/osbuild/bootc-image-builder/blob/main/bib/cmd/bootc-image-builder/main.go#L550 for an example. With this no changes to go build/go install should be necessary.